test/WebJobs.Script.Tests.Integration/Twilio/TwilioEndToEndTestsBase.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Integration.Twilio
+{
+    public class TwilioEndToEndTestsBase : IClassFixture<TwilioEndToEndTestsBase.TestFixture>
+    {
+        private EndToEndTestFixture _fixture;
+
+        public TwilioEndToEndTestsBase(TestFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        [Fact]
+        public async Task TwilioReference()
+        {
+            string testData = Guid.NewGuid().ToString();
+
+            // Run function that references Twilio Api
+            await _fixture.Host.BeginFunctionAsync("TwilioReference", testData);
+            string logs = "";
+
+            await TestHelpers.Await(() =>
+            {
+                // Wait until input has been processed
+                logs = _fixture.Host.GetLog();
+                return logs.Contains(testData);
+            });
+
+            // Make sure the input string made it all the way through
+            Assert.True(logs.Contains(testData));
```

Comment:
I noted this in the other review, but you can remove this line as the call to `Await` will fail if the testData isn't there.